things/api/things/http/endpoint_test.go

```diff
@@ -584,28 +584,28 @@ func TestViewThing(t *testing.T) {
 			id:     strconv.FormatUint(wrongID, 10),
 			auth:   token,
 			status: http.StatusNotFound,
-			res:    "",
+			res:    toJSON(errorRes{things.ErrNotFound.Error()}),
```

Comment:
This repeating of the procedure where we create error struct, then convert it to JSON is the only suspicious part to me. I have an impression that this can be achieved more elegantly. Maybe by changing `TestViewThing{}`'s `res` field type, IDK.

@nmarcetic @manuio @dusanb94 @mteodor opinions?

Reply:
@drasko I agree, if they are used in many places better create a variable for this.